_tuning-your-cluster/availability-and-recovery/remote-store/index.md

```diff
@@ -17,199 +17,77 @@ Introduced 2.10
-Remote-backed storage is enabled for an index when it is created. This feature cannot be enabled for indexes that already exist.
 
-For refresh-level durability, include the `remote_store` property to enable the feature and specify a segment repository:
+## Restoring from a backup
 
-```bash
-curl -X PUT "https://localhost:9200/my-index?pretty" -ku admin:admin -H 'Content-Type: application/json' -d'
-{
-  "settings": {
-    "index": {
-      "number_of_shards": 1,
-      "number_of_replicas": 0,
-      "replication": {
-        "type": "SEGMENT"
-      },
-      "remote_store": {
-        "enabled": true,
-        "repository": "segment-repo"
-      }
-    }
-  }
-}
-'
-```
+To restore an index from a remote backup, such as in the event of a node failure, use one of the following options:
 
-For request-level durability, in addition to the `remote_store` and segment repository, include the `translog` property and specify a translog repository:
+**Restore only unassigned shards**
 
 ```bash
-curl -X PUT "https://localhost:9200/my-index?pretty" -ku admin:admin -H 'Content-Type: application/json' -d'
+curl -X POST "https://localhost:9200/_remotestore/_restore" -H 'Content-Type: application/json' -d'
 {
-  "settings": {
-    "index": {
-      "number_of_shards": 1,
-      "number_of_replicas": 1,
-      "replication": {
-        "type": "SEGMENT"
-      },
-      "remote_store": {
-        "enabled": true,
-        "repository": "segment-repo",
-        "translog": {
-          "enabled": true,
-          "repository": "translog-repo",
-          "buffer_interval": "300ms"
-        }
-      }
-    }
-  }
+  "indices": ["my-index-1", "my-index-2"]
 }
 '
 ```
 
-You can have the same repository serve as both the segment repository and translog repository. 
-{: .note}
-
-As data is added to the index, it also will be continuously uploaded to remote storage in the form of segment and translog files because of refreshes, flushes, and translog fsyncs to disk. Along with data, other metadata files will be uploaded.
-The `buffer_interval` setting specifies the time interval during which translog operations are buffered. Instead of uploading individual translog files, OpenSearch creates a single translog file with all the write operations received during the configured interval. Bundling translog files leads to higher throughput but also increases latency. The default `buffer_interval` value is 100 ms.
-
-Setting `translog.enabled` to `true` is currently an irreversible operation.
-{: .warning}
-
-### Restoring from a backup
-
-To restore an index from a remote backup, such as in the event of a node failure, you must first close the index:
-
-```bash
-curl -X POST "https://localhost:9200/my-index/_close" -ku admin:admin
-```
-
-Restore the index from the backup stored on the remote repository:
+**Remote all shards of a given index**
 
 ```bash
 curl -X POST "https://localhost:9200/_remotestore/_restore" -ku admin:admin -H 'Content-Type: application/json' -d'
```

Comment:
To restore all the shards, we need to pass a query parameter: `restore_all_shards=true`.